db/migrate/20230907021007_change_saved_recipie_default.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeSavedRecipieDefault < ActiveRecord::Migration[7.0]
+  def change
+    ## Remove api_recipie_id default null 
+    change_column_default(:saved_recipies, :api_recipie_id, nil)
```

Comment:
❓I'm confused by this change. Can you talk me through your thought process?

Reply:
I agree, it should be default null